package.json

```diff
@@ -137,8 +139,15 @@
   "ember": {
     "edition": "octane"
   },
-  "resolutions": {
-    "@embroider/macros": "^1.0.0",
-    "ember-concurrency": "2.2.1"
+  "peerDependencies": {
+    "@babel/core": "^7.0.0",
+    "@glint/template": ">= 0.8.3",
+    "@types/node": "*",
+    "ember-cli-htmlbars": "^6.0.0"
+  },
```

Comment:
Why do you need peer dependencies? This is an app, no one would install it as a package..